<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCreator-样本S语言编译器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/logo.JPG</normaloff>:/images/logo.JPG</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>Courier New</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget_info">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_lex">
       <attribute name="title">
        <string>词法分析</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser_lex">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_parse">
       <attribute name="title">
        <string>语法分析</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser_parse">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_code">
       <attribute name="title">
        <string>中间代码</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser_code">
          <property name="font">
           <font>
            <family>Courier New</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_err">
       <attribute name="title">
        <string>错误信息</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser_err">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_msg">
       <attribute name="title">
        <string>详细信息</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser_msg">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="action_show"/>
    <addaction name="action_lex"/>
    <addaction name="action_parse"/>
    <addaction name="action_code"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_seeDoc"/>
    <addaction name="action_seeReport"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_show"/>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开一个已存在的文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存当前文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_show">
   <property name="text">
    <string>一键生成</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_parse">
   <property name="text">
    <string>语法分析</string>
   </property>
   <property name="toolTip">
    <string>查看语法分析结果</string>
   </property>
  </action>
  <action name="action_lex">
   <property name="text">
    <string>词法分析</string>
   </property>
   <property name="toolTip">
    <string>查看词法分析结果</string>
   </property>
  </action>
  <action name="action_code">
   <property name="text">
    <string>中间代码生成</string>
   </property>
  </action>
  <action name="action_11">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>汇编代码生成</string>
   </property>
  </action>
  <action name="action_12">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>创建一个新的文本文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="text">
    <string>另存为...</string>
   </property>
   <property name="toolTip">
    <string>保存到新的文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_seeReport">
   <property name="text">
    <string>检查更新</string>
   </property>
  </action>
  <action name="action_seeDoc">
   <property name="text">
    <string>说明文档</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
